### PR TITLE
added user controller and implemented methods

### PR DESCRIPTION
**Pull request by Martynice**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -48,12 +50,44 @@
...
-            <version>5.2.2.RELEASE</version>
+            <version>${spring.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-webmvc</artifactId>
+            <version>${spring.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
```
Comment (jah701) [inline pom.xml:66]: I'm not sure if we need jsp, jstl and servlet dependencies for this h\w

File: src/main/java/com/dao/UserDaoImpl.java [hunk 2]
```diff
@@ -40,11 +39,20 @@ public void add(User user) {
         }
     }
 
+    @Override
+    public User get(Long id) {
```
Comment (jah701) [inline src/main/java/com/dao/UserDaoImpl.java:43]: Maybe it is better to return Optional<User> from DB

**Review by jah701 — APPROVED** [review]
Well done!

File: src/main/java/com/config/WebConfig.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"com.controllers"})
+public class WebConfig {
+    @Bean
+    public InternalResourceViewResolver resolver() {
```
Comment (RomanDubovskyi) [inline src/main/java/com/config/WebConfig.java]: second point 
https://mate-academy.github.io/jv-program-fulltime/04_spring/checklist/hw_32.html

File: src/main/java/com/dao/UserDaoImpl.java (hunk 2 above)
Comment (RomanDubovskyi) [reply src/main/java/com/dao/UserDaoImpl.java:43]: > Maybe it is better to return Optional from DB

in this case, it's not obligatory

Comment (RomanDubovskyi) [inline src/main/java/com/dao/UserDaoImpl.java:43]: let's use session get here, that will be shorter

Comment (Martynice) [reply src/main/java/com/dao/UserDaoImpl.java:43]: done

File: src/main/java/com/config/WebConfig.java (hunk 3 above)
Comment (Martynice) [reply src/main/java/com/config/WebConfig.java]: done

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**